### PR TITLE
Rename outputs MQTT topics

### PR DESCRIPTION
**Pull request by samuel-rufi**
Fixes https://github.com/iotaledger/iota.rs/issues/1033

### CODE REVIEW

File: src/node_api/mqtt/types.rs [hunk 1]
```diff
@@ -171,9 +171,9 @@ impl Topic {
             r"^transactions/0x([a-f0-9]{64})/included-block$",
             // Output topics
             r"^outputs/([a-f0-9]{64})(\d{4})$",
```
Comment (thibault-martinez) [inline src/node_api/mqtt/types.rs:173]: This one doesn't have a `basic` prefix like the core API?

Comment (samuel-rufi) [reply src/node_api/mqtt/types.rs:173]: Nope there is no prefix here <https://studio.asyncapi.com/?url=https://raw.githubusercontent.com/iotaledger/tips/stardust-event-api/tips/TIP-0028/event-api.yml>

Comment (samuel-rufi) [reply src/node_api/mqtt/types.rs:173]: But the Core API doesn't have a prefix too, only the Indexer API

Comment (thibault-martinez) [reply src/node_api/mqtt/types.rs:173]: I meant the indexer sorry

Comment (samuel-rufi) [reply src/node_api/mqtt/types.rs:173]: Hmm not sure now if this MQTT endpoint now also listens for outputs that are not "basic"

Comment (Thoralf-M) [reply src/node_api/mqtt/types.rs:173]: In https://github.com/iotaledger/inx-mqtt/commit/c06f2a2fffc0a825b06d8be38c1c874eb8893c7e it didn't get changed

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

File: src/node_api/mqtt/types.rs (hunk 1 above)
Comment (samuel-rufi) [reply src/node_api/mqtt/types.rs:173]: okay let's keep it then

**Pull request merged**